moduleframework/tools/static_dockerfile_checks.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import print_function
+
+import os
+from avocado import Test
+from moduleframework import module_framework
+from moduleframework import dockerlinter
+
+class DockerInstructionsTests(module_framework.AvocadoTest):
```

Comment:
you've forgot  to change ``module_framework.AvocadoTest`` to ``Test``

Reply:
I use `assert_to_warn` function. AvocadoTest is needed. I don't see any reason, why not use AvocadoTest.